synapse/lib/hiveauth.py

```diff
@@ -841,6 +841,36 @@ async def grant(self, roleiden, indx=None):
 
         await self.auth.setUserInfo(self.iden, 'roles', roles)
 
+    async def grants(self, roleidens):
```

Comment:
If i had it to do again, i'd probably name the `.grant()` API above `addRole()`.  Lets dig up and call this guy `setRoles()`